modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/LanguageServiceEndpoint.java

```diff
@@ -65,6 +72,22 @@ void setLanguageService(LanguageService languageSrv) {
     this.languageSrv = languageSrv;
   }
 
+  private static Set<String> excludedLocales;
+
+  public static final String EXCLUDE_CONFIG_KEY = "org.opencastproject.adminui.languages.exclude";
+
+  /** OSGi callback if properties file is present */
+  @Override
+  public void updated(Dictionary<String, ?> properties) throws ConfigurationException {
+    if (properties == null) {
+      logger.info("No configuration available, using defaults");
```

Comment:
This is not correct. Deleting the configuration file at runtime would cause `properties == null` but `excludedLocales` would still be set to the old value. Initializing the set right at the beginning of the method would solve the issue:

```java
excludedLocales = new HashSet<>();
if (properties == null) {
  …
```